### ISSUE TITLE
Weights are in tar format, not gzip

### ISSUE THREAD

**Issue by nkconnor** [allenai/deepfigures-open#1]
https://s3-us-west-2.amazonaws.com/ai2-s2-research-public/deepfigures/weights.tar.gz

Since this is a tar file, it would be helpful to fix the file name or provide a hint to this in the README

**Comment by tu-cao** [comment]
Well, I think it is only .tar.

Therefore, You just do `tar -xf <file>` for decompressing it.

**Comment by nkconnor** [comment]
Yes exactly, that's why the file should be called `.tar` not `.tar.gz`.
